app/assets/stylesheets/components/common/_newsletter_signup.scss

```diff
@@ -6,6 +6,11 @@
   @include body(16, 22);
 }
 
+.t-newsletter-strap {
```

Comment:
`t-` classes are typically used for testing purposes only. Is that what this is?